### ISSUE TITLE
Topics page lists modules but no descriptions

### ISSUE THREAD

**Issue by aslze** [doxygen/doxygen#10904]
**Describe the bug**

Groups in my project appear in my navbar and clicking on their heading opens the Topics page (formerly "Modules").

That page used to list the topics in a table with a short description of each (taken from the first line of the Group description text under `\defgroup`.

But now that page only lists the topic names. The right column is empty.

**Screenshots**
This is the Topics page (on 1.11.0)

![imagen](https://github.com/doxygen/doxygen/assets/36175101/d588d5d7-095e-43af-8999-23767b8697cd)

**To Reproduce**
Sample micro project:
[sample.zip](https://github.com/doxygen/doxygen/files/15476200/sample.zip)

**Expected behavior**
There should be a table with names and descriptions. That used to be with doxy 1.9.x (I did not try 1.10.x).

**Version**
1.11.0

**Comment by aslze** [comment]
Same happens in version 1.10.0. But in 1.9.1 the descriptions were shown.

This is the page created by doxy 1.9.1:
![imagen](https://github.com/doxygen/doxygen/assets/36175101/823c856c-45eb-41a2-8d5b-93a9825faf76)

BTW, one change between versions is that until 1.9.1 that page was called "Modules" and now it is "Topics" (that is the default title and the type name in the XML layout file).

**Comment by albert-github** [comment]
> BTW, one change between versions is that until 1.9.1 that page was called "Modules" and now it is "Topics" (that is the default title and the type name in the XML layout file).

This is correct, this had to be done due to the introduction of C++20 modules (in doxygen version 1.9.8) and it has as benefit that the already confusing "Modules" is now a bit clearer.

The problem has been introduced by doxygen version 1.10.0

**Comment by albert-github** [comment]
I've just pushed a proposed patch, pull request #10905

**Comment by albert-github** [comment]
Code has been integrated in master on GitHub (please don't close the issue as this will be done at the moment of an official release).

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.12.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by RobertWCarey** [comment]
**Describe the bug**
I came across this issue after updating from `1.9.4` to `1.12.0` (at least it looks a lot like it). Running `1.12.0` over the sample project provided in the [original comment](https://github.com/doxygen/doxygen/issues/10904#issue-2321955202) I see the same behavior. I.e. no description next to the topic. If anything I think it might've gotten worse (compared to `1.11.0`) as now `@brief` commands seem to be affected :see_no_evil:.

Playing around a little more it seems like it is related to setting/using `JAVADOC_AUTOBRIEF=YES` and having a blank line before the first sentence. If there is no blank line then the description shows up as expected. However, with a blank line it doesn't appear. With `1.11.0` it seems like this was only the case for description with no `@brief` command but in `1.12.0` it looks like it now affects descriptions with an `@brief` command.

> Note that the blank line before the `@brief` line does not appear to cause issues when `JAVADOC_AUTOBRIEF=NO`

**Screenshots**
_1.9.4_
![image](https://github.com/user-attachments/assets/80248b9d-bbac-4405-b444-474539bebb53)

_1.11.0_
![image](https://github.com/user-attachments/assets/2390ae98-7a76-40a8-b4d6-bdbc4f55f051)

_1.12.0_ 
![image](https://github.com/user-attachments/assets/d76465bf-eb6a-423c-9710-fdde74ab893a)

**To Reproduce**
[sample_241108.zip](https://github.com/user-attachments/files/17669913/sample_241108.zip)

**Expected behavior**
Descriptions to appear regardless of blank lines in between the brief and the defgroup.

**Version**
```
➜   /opt/doxygen-1.12.0/bin/doxygen --version            
1.12.0 (c73f5d30f9e8b1df5ba15a1d064ff2067cbb8267)

➜   /opt/doxygen-1.11.0/bin/doxygen --version            
1.11.0 (9b424b03c9833626cd435af22a444888fbbb192d)

➜   /opt/doxygen-1.9.4/bin/doxygen --version            
1.9.4 (5d15657a55555e6181a7830a5c723af75e7577e2)
```

**Comment by albert-github** [comment]
I tested it with the doxygen 1.12.0 version and got the same results as you got.
I tested with the current master (1.13.0 (47bd622d96671a2cb0864f3a164723ec7bcbe2bf)) and got again the results as you showed with the doxygen 1.9.4 version.

I think the problem has been fixed but not released.

**Comment by doxygen** [comment]
This issue was previously marked 'fixed but not released',
which means it should be fixed in doxygen version 1.13.0.
Please verify if this is indeed the case. Reopen the
issue if you think it is not fixed and please include any additional information
that you think can be relevant (preferably in the form of a self-contained example).

**Comment by RobertWCarey** [comment]
Tested with 1.13.0 and it now behaves as expects 🎉.
